src/setparam.f90

```diff
@@ -516,6 +516,10 @@ module xtb_setparam
 !! ------------------------------------------------------------------------
    !> PTB settings
    type(TPTBSetup) :: ptbsetup
+   !> GFN-FF manual setup of nb list via xcontrol
+   !  allows a maximum of 164 atoms neighbors to be changed
+   !  ffnb(42,i) stores the number of neighbors of atom i
+   integer :: ffnb(42,164) = -1
```

Comment:
Do you see any chance to make the size of this array dependent on the input (by scanning the input first and then allocating the required memory)? If not, this is still a valid approach, but we should at least document why it's not reasonable and we take the approach of a fixed memory variable.

Reply:
Let's shift it to another PR, @Thomas3R.